TRACKING.md

```diff
@@ -8,9 +8,9 @@ When a store opts in to WooCommerce usage tracking and uses _Pinterest for WooCo
 
 As in WooCommerce core, only non-sensitive data about how a store is set up and managed is tracked. We **do not track or store personal data** from your clients.
 
-<woo-tracking-jsdoc>
+<woocommerce-grow-tracking-jsdoc>
```

Comment:
❔

Not related to Pinterest itself, more related to `woocommerce-grow-jsdoc` 😄  wouldn't it be better to use a more generic name instead of `woocommerce-grow-tracking-jsdoc`? Something like `<tracking-jsdoc>`? I am just thinking that maybe other divisions or external people want to use this package outside of Grow and they would like to have something more generic. It is not an issue of course but more a curiosity.

Reply:
We briefly discussed it at https://github.com/woocommerce/grow/pull/6#pullrequestreview-982416113 and I created a somewhat related issue in https://github.com/woocommerce/grow/issues/7

I think once, we decide it's ready for publishing to the outer world, we could change that. To me one of the purposes of `/grow` is to incubate tools before proposing to external people. 

Before that, I think it would be nice to have that explicit distinction, between which things are already woo-generic, and which are our incubated things.

However, it would require tedious work to change that later on, and having it generic and short from day one could give more benefits.

------

Speaking of `<tracking-jsdoc>` the good practice for custom element names is to prefix them with the vendor. Even though technically this is not a custom element per se, just a custom named `HTMLElement` `(`HTMLUnknownElement`). But IMHO it's good to stick to that practice, as we may someday create a custom element for that that could do some work in the browser.

Personally, I'd rather shorten this to `<woo-tracking-jsdoc>`/`<wc-tracking-jsdoc>`

Reply:
Thanks, @tomalec for your detailed explanation! 😄